internal/cli/kraft/net/remove/remove.go

```diff
@@ -65,6 +68,29 @@ func (opts *RemoveOptions) Run(ctx context.Context, args []string) error {
 		return err
 	}
 
+	machineController, err := mplatform.NewMachineV1alpha1ServiceIterator(ctx)
+	if err != nil {
+		return err
+	}
+
+	machines, err := machineController.List(ctx, &machineapi.MachineList{})
+	if err != nil {
+		return err
+	}
+
+	for _, machine := range machines.Items {
+		if machine.Status.State != machineapi.MachineStateRunning {
+			continue
+		}
+		for _, network := range machine.Spec.Networks {
+			if network.IfName == args[0] {
```

Comment:
It'd be good to ensure the length of args somewhere before this.

Reply:
Nice catch, thanks for this! I added an additional check.